docs/project-list.md

```diff
@@ -0,0 +1,103 @@
+---
+date-generated: 2020-03-30T16:36:02
```

Comment:
Did you mean to have 2 files, project-list and project-overview?
I think they are probably best suited as 1.

Reply:
I actually just ran MSBuild and that was the output.
@dominoFire did I do something wrong?

Reply:
You can delete this file. It should be docs\project-overview.md, as shown below:

https://github.com/NuGet/NuGet.Client/blob/67f7f1bc0fc40fbfca6453c9bb635540ae29732c/build/docs.proj#L50-L51